### PR TITLE
[testing] [ci]  add deckhouse logs if not READY pod

### PR DESCRIPTION
**Pull request by alexey-gavrilov-flant**
## Description
Adds Deckhouse logs if Pod hasn't changed to ready status.

## Why do we need it, and what problem does it solve?
We cannot get deckhouse logs when e2e test was failed.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Manual testing proofs
Negative (e2e tests was failed) tested [here](https://github.com/deckhouse/deckhouse-test-2/pull/326)
Positive (e2e tests was passed) [here](https://github.com/deckhouse/deckhouse-test-2/pull/327)

## Changelog entries

```changes
section: ci
type: fix 
summary: Added deckhouse logs if the Pod hasn't changed to ready status.
impact_level: low
```

### CODE REVIEW

File: testing/cloud_layouts/script.sh [hunk 1]
```diff
@@ -646,6 +650,8 @@ END_SCRIPT
       >&2 echo -n "  Deckhouse Pod not ready. Attempt $i/$testRunAttempts failed."
     fi
   done
+  >&2 echo -n "Fetch Deckhouse logs if error test ..."
+  $ssh_command -i "$ssh_private_key_path" $ssh_bastion "$ssh_user@$master_ip" sudo su -c /bin/bash > "$cwd/deckhouse.json.log" <<<"${testLog}"
```
Comment (diafour) [inline testing/cloud_layouts/script.sh:654]: I suggest to no use $cwd for logging and put deckhouse.json.log into subdir of /tmp/ (e.g. /tmp/deckhouse), and upload it with this line:

```
            ${{ steps.setup.outputs.tmp-dir-path}}/deckhouse
```

Comment (alexey-gavrilov-flant) [reply testing/cloud_layouts/script.sh]: added /tmp/logs/

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @alexey-gavrilov-flant. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @alexey-gavrilov-flant. I've started the workflow for label 'e2e/run/openstack'...

**Comment by name212** [comment]
We already have deckhouse logs in the [artifact](https://github.com/deckhouse/deckhouse/blob/c48e346c5509ca4520312d1d764a62dc2f3c98a2/testing/cloud_layouts/script.sh#L688)

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @alexey-gavrilov-flant. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Review by name212 — APPROVED** [review]

**Pull request merged**